install_script.sh.template

```diff
@@ -74,13 +74,30 @@ function report(){
 }
 
 function report_telemetry() {
+  local install_id="$1"
+  local install_type="$2"
+  local install_time="$3"
+
   if [ "$DD_INSTRUMENTATION_TELEMETRY_ENABLED" == "false" ] || \
     [ "$site" == "ddog-gov.com" ] || \
     [ -z "${apikey}" ] || \
     [ -z "$telemetry_url" ]; then
     return
   fi
 
+  install_id_tag=
+  install_type_tag=
+  install_time_tag=
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ These are instantiated empty because we're always injecting them into the tags?